src/util.cc

```diff
@@ -100,11 +102,26 @@ static void MakeUtf8String(Isolate* isolate,
                            MaybeStackBuffer<T>* target) {
   Local<String> string;
   if (!value->ToString(isolate->GetCurrentContext()).ToLocal(&string)) return;
+  String::ValueView value_view(isolate, string);
+
+  if (value_view.is_one_byte() && simdutf::validate_ascii(reinterpret_cast<const char*>(value_view.data8()),value_view.length()) ) {
```

Comment:
Possibly worth it to store value_view.length() in a variable as it's being called multiple times.

Reply:
agreed.